### PR TITLE
republishedexistingmodelswithhftransformerflavour-v2

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
republishedexistingmodelswithhftransformerflavour-v2
databricks-dolly-v2-12b
bert-base-cased
deepset-minilm-uncased-squad2
bert-base-uncased
microsoft-deberta-large
roberta-base-openai-detector
gpt2
microsoft-deberta-base
distilgpt2

**databricks-dolly-v2-12b**  
(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)

(Deployment component ran separately and it was successful) 
(Tested fine tune token-classification but failed at compute metrics as it was issue to be fixed in fine tune as instructed Raised PR)
 [https://ml.azure.com/experiments/id/f868cd36-9ea7-400f-8830-5067656cb448/runs/dc71b237-96ed-4308-98e0-dc62dfdb37c6?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**bert-base-cased** 
(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)

[https://ml.azure.com/experiments/id/542b7761-3490-43ef-b97b-d260a6c7c107/runs/yellow_flag_n5xrvljd64?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**deepset-minilm-uncased-squad2**
(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)

[https://ml.azure.com/experiments/id/d1b7edf7-e95a-4426-966e-b58c25480f56/runs/maroon_receipt_8hrz2lsyc9?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**bert-base-uncased**
(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)

[https://ml.azure.com/experiments/id/542b7761-3490-43ef-b97b-d260a6c7c107/runs/crimson_beard_3q8zp599xx?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**microsoft-deberta-large**
Job Successful from import to batch deploy.

[https://ml.azure.com/experiments/id/2bb40d8e-7c40-466a-af5e-1d9b56cb18d0/runs/musing_brain_nmn9x45dh7?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**roberta-base-openai-detector**
(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)
failed at profile endpoint due to unavailability of end point this is caused because due to quota issues we deleted end points.

[https://ml.azure.com/experiments/id/daf7fc8a-4545-4911-99d1-a04fe5131a50/runs/sharp_milk_81m63rthmv?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**gpt2**

(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)

(Fine tune token-classification failed at compute metrics as it was issue to be fixed in fine tune as instructed Raised PR)

[https://ml.azure.com/experiments/id/c6724364-c89d-4523-bdfe-b77bf36989e3/runs/bold_gold_rtv99bq5vp?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**microsoft-deberta-base**
Job Successful from import to batch deploy.

[https://ml.azure.com/experiments/id/18f11c6f-c6c6-4e6d-a473-77e725e90d97/runs/802d2849-64e0-460b-b099-a315e42eed4d?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**distilgpt2**

(For this PR request we did not consider DSMII and tgi deployment components because of quota issue)

(Fine tune token-classification failed at compute metrics as it was issue to be fixed in fine tune as instructed Raised PR)

[https://ml.azure.com/experiments/id/0f020411-bbcd-4256-adcb-b55ba303a32a/runs/tough_bean_5mkjb6xkcx?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

### CODE REVIEW

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 1]
```diff
@@ -16,5 +16,10 @@ tags:
   - Standard_ND40rs_v2
   - Standard_ND96asr_v4
   license: mit
+  model_specific_defaults:
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:19]: Are these settings needed for deployment?

Comment (KonakanchiSwathi) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml:19]: Yes Sasi

Comment (skasturi) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml:22]: @ShipraJain01 Are the "model_specific_defaults" settings used for evaluation?

File: assets/models/system/bert-base-cased/description.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Fill Mask|Fill Mask|<a href="https://huggingface.co/datasets/rcds/wikipedia-for-
 #### Sample input
 ```json
```
Comment (novaturient95) [inline assets/models/system/bert-base-cased/description.md:32]: most run links in description are showing not found for me. Are they deleted?

**Review by novaturient95 — DISMISSED** [review]

File: assets/models/system/bert-base-cased/description.md (hunk 2 above)
Comment (KonakanchiSwathi) [reply assets/models/system/bert-base-cased/description.md:32]: databricks-dolly-v2-12b
[https://ml.azure.com/experiments/id/9ba24919-1701-4f71-a6a6-beaaf5960adf/runs/amiable_snail_nn94rl49v0?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
bert-base-cased
[https://ml.azure.com/experiments/id/46470d27-0d00-4f93-8e1b-ef828f0e7f83/runs/boring_egg_6cg3trjkcc?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
deepset-minilm-uncased-squad2
[https://ml.azure.com/experiments/id/7e4b5293-1f32-40f3-b1e1-0692269cbcce/runs/honest_kale_2pvnb8k82p?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
bert-base-uncased
[https://ml.azure.com/experiments/id/c2ebc12b-61bf-415f-979e-5844ca17a0ed/runs/sleepy_pencil_7k7f3wcfd7?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
microsoft-deberta-large
[https://ml.azure.com/experiments/id/1f813df6-5ec0-4162-aaa7-4e0063d29354/runs/epic_seed_8tk0tjbntm?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
roberta-base-openai-detector
[https://ml.azure.com/experiments/id/80473057-1ce3-4524-8e25-f23259b42537/runs/icy_gas_f5d3tt9x6c?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
gpt2
[https://ml.azure.com/experiments/id/57df0b68-cc10-4678-81f0-9a0eddb99044/runs/sad_wall_klg7wf0bjn?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
microsoft-deberta-base
[https://ml.azure.com/experiments/id/460e3a32-9d5c-44fc-a195-927253a4d539/runs/ashy_holiday_3h8vtqbwc5?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
distilgpt2
[https://ml.azure.com/experiments/id/a1c138b0-806b-4edd-9a24-59385a06bf86/runs/strong_chaconia_bvxxk2v7m1?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

Comment (KonakanchiSwathi) [reply assets/models/system/bert-base-cased/description.md:32]: Added

File: assets/models/system/databricks-dolly-v2-12b/description.md [hunk 3]
```diff
@@ -36,11 +36,23 @@ Standard_NC24s_v3 | 1.08 | 34.9
 }
 ```
 
+#### Sample output
+```json
+{
+    "input_data": {
+        "input_string": ["My name is John and I am", "Once upon a time,"]
```
[the remark this answers is not in the document]
Comment (KonakanchiSwathi) [reply assets/models/system/databricks-dolly-v2-12b/description.md:49]: Done Added and comitted

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 4]
```diff
@@ -13,8 +13,16 @@ tags:
   Featured: ''
   Preview: ''
   computes_allow_list:
+  - Standard_NC12s_v3
```
[the remark this answers is not in the document]
Comment (KonakanchiSwathi) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml]: Added and updated

File: assets/models/system/bert-base-cased/description.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ Fill Mask|Fill Mask|<a href="https://huggingface.co/datasets/rcds/wikipedia-for-
 #### Sample input
```
[the remark this answers is not in the document]
Comment (KonakanchiSwathi) [reply assets/models/system/bert-base-cased/description.md:31]: databricks-dolly-v2-12b
[[save_model_to_blobstorage databricks-dolly-v2-12b - Azure AI | Machine Learning Studio](https://ml.azure.com/experiments/id/9ba24919-1701-4f71-a6a6-beaaf5960adf/runs/amiable_snail_nn94rl49v0?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)](url)
bert-base-cased
[https://ml.azure.com/experiments/id/46470d27-0d00-4f93-8e1b-ef828f0e7f83/runs/boring_egg_6cg3trjkcc?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
deepset-minilm-uncased-squad2
[https://ml.azure.com/experiments/id/7e4b5293-1f32-40f3-b1e1-0692269cbcce/runs/honest_kale_2pvnb8k82p?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
bert-base-uncased
[https://ml.azure.com/experiments/id/f4dc98b5-d1c0-45ec-a32e-86347e8cb166/runs/sharp_branch_q5jhhvq7mb?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
microsoft-deberta-large
[https://ml.azure.com/runs/epic_seed_8tk0tjbntm?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourcegroups/shared-finetuning-rg/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
roberta-base-openai-detector
[https://ml.azure.com/experiments/id/80473057-1ce3-4524-8e25-f23259b42537/runs/icy_gas_f5d3tt9x6c?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
gpt2
[https://ml.azure.com/experiments/id/57df0b68-cc10-4678-81f0-9a0eddb99044/runs/sad_wall_klg7wf0bjn?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)
microsoft-deberta-base
[https://ml.azure.com/experiments/id/460e3a32-9d5c-44fc-a195-927253a4d539/runs/ashy_holiday_3h8vtqbwc5?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47](url)
distilgpt2
[https://ml.azure.com/experiments/id/a1c138b0-806b-4edd-9a24-59385a06bf86/runs/strong_chaconia_bvxxk2v7m1?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#](url)

**Comment by KonakanchiSwathi** [comment]
Reran all components and up dated

**Pull request closed without merging**